pytensor/tensor/sort.py

```diff
@@ -1,25 +1,32 @@
+from typing import Literal
+
 import numpy as np
 
 from pytensor.gradient import grad_undefined
-from pytensor.graph.basic import Apply, Constant
+from pytensor.graph.basic import Apply
 from pytensor.graph.op import Op
 from pytensor.misc.safe_asarray import _asarray
 from pytensor.tensor.basic import arange, as_tensor_variable, switch
-from pytensor.tensor.math import eq, ge, mul
+from pytensor.tensor.math import eq, ge
 from pytensor.tensor.type import TensorType
 
 
-def _variable_is_none(var):
-    return isinstance(var, Constant) and var.data is None
+KIND_VALUES = ["quicksort", "mergesort", "heapsort", "stable"]
+KIND = Literal["quicksort", "mergesort", "heapsort", "stable"]
```

Comment:
```suggestion
KIND = typing.get_args(KIND_VALUES)
```

This way there's no need to keep the two lists synchronized if they get changed in the future

Reply:
Your suggestion did not work, did you mean something slightly different?

Reply:
Yes, sorry. Here's what I have in mind:

```
KIND= Literal["quicksort", "mergesort", "heapsort", "stable"]
KIND_VALUES: tuple[KIND, ...] = typing.get_args(KIND)
```